client/my-sites/email/inbox/mailbox-selection-list/style.scss

```diff
@@ -0,0 +1,196 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+body {
+	background-color: var( --studio-white );
+}
+
+.mailbox-selection-list {
+
+	.mailbox-selection-list__header {
+		.formatted-header__title {
+			font-size: $font-title-large;
+			line-height: $font-headline-medium;
+
+			@include break-large {
+				font-size: $font-headline-medium;
+			}
+		}
+
+		.formatted-header__subtitle {
+			font-size: $font-body;
+			line-height: $font-title-medium;
+		}
+	}
+
+	.mailbox-selection-list__new-mailbox-upsell-container {
+		background: #f9f9f9;
```

Comment:
@saygunnyc, could you please review the colours here to make sure we don't have standard colours that we can use? I am very wary of introducing new, hard-coded colours local to these screens. It's possible these colours are from WordPress core, but it would be good to confirm that.